jetty-ee9/jetty-ee9-proxy/src/test/java/org/eclipse/jetty/ee9/proxy/ProxyServletTest.java

```diff
@@ -197,16 +201,18 @@ private void startClient() throws Exception
 
     private void startClient(Consumer<HttpClient> consumer) throws Exception
     {
-        client = prepareClient(consumer);
+        clientProxies.add(new HttpProxy("localhost", proxyConnector.getLocalPort()));
+        client = prepareClient(consumer, clientProxies);
```

Comment:
Just initialize the `proxy` field, no need to pass it around, as `prepareClient()` will have access to it.